src/pudl/analysis/pudl_models.py

```diff
@@ -120,5 +120,13 @@ def out_sec10k__parents_and_subsidiaries() -> pd.DataFrame:
         "standard_industrial_classification"
     ].str.extract(r"(.+)\[(\d{4})\]")
     df["industry_id_sic"] = df["industry_id_sic"].astype("string")
+    # Some utilities harvested from EIA 861 data that don't show up in our entity
+    # tables. These didn't end up improving coverage, and so will be removed upstream.
+    # Hack for now is to just drop them so the FK constraint is respected.
+    # See https://github.com/catalyst-cooperative/pudl/issues/4050
+    bad_utility_ids = [
+        3579,  # Cirro Group, Inc. in Texas
+    ]
```

Comment:
Probably this is a rogue `utility_id_eia` that only exists in the EIA-861 data, and it should go away if we remove the experimental EIA-861 ID harvesting upstream, as @katie-lamb said she intends to.